src/docfx/build/xref/ExternalXrefMapLoader.cs

```diff
@@ -115,19 +124,29 @@ private static List<(string uid, long start, long end)> GetXrefSpecPositions(Rea
                         {
                             stack.Push((Encoding.UTF8.GetString(reader.ValueSpan), top.start));
                         }
+                        if (propertyNames.Length > 0 && stack.Count == 1)
+                        {
+                            foreach (var pname in propertyNames)
+                            {
+                                if (reader.ValueTextEquals(Encoding.UTF8.GetBytes(pname)) && reader.Read() && reader.TokenType == JsonTokenType.String)
```

Comment:
nit; extract `Encoding.UTF8.GetBytes(pname)` just like `s_uidBytes` if this method returns `repositoryUrl`.

Reply:
OK.